static/src/js/util/map/normalizeGranuleSpatial.js

```diff
@@ -193,6 +193,8 @@ const normalizeGranuleSpatial = (granule) => {
         })))
 
         dividedCoordinates.forEach((p) => {
+          if (p.length < 3) return
```

Comment:
So this is a bug on edsc-utils right? It shouldn't be giving us polyons with < 3 sides? Shall I write a ticket?

Reply:
Yes it is a bug in geo utils, no don't write a ticket